src/test/java/io/confluent/connect/hdfs/wal/WALTest.java

```diff
@@ -73,6 +73,8 @@ public void run() {
     });
     thread.start();
 
+    // AcquireLease will try to acquire the same lease that wal1 is holding and fail
```

Comment:
nit: I'd use the actual method name (`acquireLease`) if I wanted to refer to it, instead of the concept ("Acquiring the lease")